deployment/elasticsearch/elasticsearch.go

```diff
@@ -339,12 +339,15 @@ func (c *Client) IndicesRecovery(indices []string) ([]IndexShardRecovery, error)
 		return nil, fmt.Errorf("unable to perform IndicesRecovery request: %w", err)
 	}
 
-	recovery := []IndexShardRecovery{}
+	recovery := []SnapshotIndexShardRecovery{}
 	for _, resp := range indicesRecovery {
 		for _, shard := range resp.Shards {
-			recovery = append(recovery, IndexShardRecovery{
+			// Add only the shards correspondign to the snapshot restoration
```

Comment:
typo at corresponding

Reply:
Thanks! Fixed in https://github.com/mattermost/mattermost-load-test-ng/pull/739/commits/f8275844cc4dfaf306389e6943c7214463545f6c